docs/cedarling/cedarling-jwt-validation.md

```diff
@@ -1,71 +1,191 @@
+  cedarling.authorize({
+    "tokens": {
+      "access_token": "<access_token>", // will be validated and used for entity creation
+      "id_token": "<id_token>",         // will be ignored unless defined in token_metadata
+    },
+    // ...
+  })
+  ```
+
+## JWT Content Validation
+
+Cedarling also supports validating the contents of a JWT by enforcing the presence of required claims. These requirements are defined in the `token_metadata` section of the policy store.
+
+### Required Claims
+
+If timestamps are provided in the context, Cedarling always verifies: 
 
 - `exp` (expiration)
 - `nbf` (not before)
 
-If the Cedarling property `CEDARLING_ID_TOKEN_TRUST_MODE` is `Strict`, the Cedarling will:
+### Custom Required Claims
 
-- Discard `id_token` if the `aud` claim does not contain the `client_id` of the access token
-- Discard Userinfo tokens that are not associated with a `sub` claim from the `id_token`
+You can specify additional required claims in your token metadata configuration:
 
+```json
+{
+  // ... other fields have been omitted for brevity
+  "trusted_issuers": {
+    "my_trusted_issuer_id": {
+      "name": "my trusted issuer",
+      "description": "an IDP that i trust",
+      "openid_configuration_endpoint": "https://test.issuer.com/.well-known/openid-configuration",
+      "token_metadata": { 
+        "access_token": {
+          "entity_type_name": "Jans::Access_token",
+          "required_claims": ["exp", "client_id"]
+        }
+      }
+    }
+  }
+}
+```
 
-### JWT Status Validation
+The above configuration means that any `access_token` must contain both the `exp` and `client_id` claims, or it will be rejected. Additionally, *registered claims*[^registered-claims] like the `exp` will also be validated according to RFC 7519 standards.
 
-The Cedarling can also check for JWT revocation if you have the `CEDARLING_JWT_STATUS_VALIDATION` property `Enabled`. The Cedarling checks the status bit of the Status Token JWT, as described in the [OAuth Status Lists](https://datatracker.ietf.org/doc/draft-ietf-oauth-status-list/) draft. Token status enforcement mitigates account takeover by enabling immediate revocation of all tokens issued to an attacker. 
+[^registered-claims]: Registered claims like `exp`, `iat`, `iss`, etc., are defined in [`RFC 7519, Section 4.1`](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1)
 
 ## ID Token Trust Mode
 
-The level of validation for the ID Token JWT can be set to either `None` or `Strict`.
+Cedarling supports an optional strict trust mode for validating relationships between different token types—primarily ID tokens, Access tokens, and Userinfo tokens.
+
+This behavior is controlled via the `CEDARLING_ID_TOKEN_TRUST_MODE` [bootstrap property](#related-bootstrap-properties).
+
+> **Implementation Status:** Currently, only `strict` and `never` modes are implemented. Additional modes (`always`, `ifpresent`) are defined in the enum but not yet supported in the validation logic.
```

Comment:
https://docs.jans.io/head/cedarling/cedarling-properties/ this file should be updated with this information, that `always` and `ifpresent` are not yet implemented.